dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -221,7 +224,10 @@ def get_torchbench_gpu_config(
 
   set_up_cmds = set_up_torchbench_gpu(model_name, nvidia_driver_version)
   local_output_location = "/tmp/xla/benchmarks/output/metric_report.jsonl"
-  gcs_location = f"{gcs_bucket.BENCHMARK_OUTPUT_DIR}/torchbench_config/{accelerator_type}/{int(datetime.datetime.now().timestamp())}/metric_report_gpu.jsonl"
+  # The gcs location can be overwritten if we use run_with_gcs_name_generatioin().
+  gcs_location = (
```

Comment:
As we will directly use the passed in location in post_metric, do we still need this to construct the `metric_config.MetricConfig`. If not, can we remove them here?